airbyte-oauth/src/main/java/io/airbyte/oauth/flows/ZendeskTalkOAuthFlow.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.oauth.flows;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.config.persistence.ConfigRepository;
+import io.airbyte.oauth.BaseOAuth2Flow;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import java.util.function.Supplier;
+import org.apache.http.client.utils.URIBuilder;
+
+/**
+ * Following docs from
+ * https://support.zendesk.com/hc/en-us/articles/4408845965210-Using-OAuth-authentication-with-your-application
+ */
+public class ZendeskTalkOAuthFlow extends BaseOAuth2Flow {
```

Comment:
Please add tests for this new class.

Reply:
As I understand we have Java team that helps us with Java tests. After merge to be done they will do a test for the class. Am I right? @lazebnyi @sherifnada